tests/unit/test.utils.js

```diff
@@ -602,11 +603,37 @@ describe('basicCompatVersionComparison', () => {
     expect(basicCompatVersionComparison('61', 60)).toBe(true);
   });
 
+  it('should return false when version added is equals than min version', () => {
+    expect(basicCompatVersionComparison('61.5.2', 61)).toBe(false);
```

Comment:
This makes me think that we have a bug in the current comparison function. I haven't looked in depth yet but I am wondering why we wouldn't allow this..

Reply:
I took another look and this `basicCompatVersionComparison` function seems to do the opposite of what I initially thought.